db/schema.rb

```diff
@@ -14,7 +14,6 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "citext"
   enable_extension "pg_stat_statements"
-  enable_extension "pgcrypto"
```

Comment:
I think this is left over from https://github.com/18F/identity-idp/pull/5275/files#diff-51a7a6164b57d05d4ec51d2c66b5fc9abb235f36f41ce2188cf9ab13c15bd256R4 since the [GoodJob migration
](https://github.com/18F/identity-idp/blob/4d88aa7/db/worker_jobs_migrate/20210809201322_create_good_jobs.rb#L4) was originally in this folder.